[added + fixed] More website details and fixes to accessibility

DylanDevelops opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have tested my changes locally and they work
- [x] My pull request targets the `main` branch of this repository.
- [x] I have organized any project files to the best of my ability

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #30

<!-- Feel free to add any additional description of changes below this line -->
This pull request includes several changes to improve the visual presentation and user experience of the website. The most important changes involve enhancing image handling across various components, adding decorative images, and improving text styling.

### Image Handling Enhancements:
* [`app/(main)/_components/main-content.tsx`](diffhunk://#diff-bca88fd766f976eac9a0531fdae69d12d05f26bb6ba8f820aaa45b94d2037427R21-R28): Added an `Image` component with a main background image to `MainContent`.
* [`app/(main)/_components/track.tsx`](diffhunk://#diff-0854ade927e647b49f2178f6a1a12e7a85feda6ebd6727e1764156206acec622L7-R27): Updated `Track` component to use different images for desktop and mobile views, and added `select-none` and `draggable={false}` attributes to images.
* [`components/navbar.tsx`](diffhunk://#diff-697415f51799297bd64388e550f60789de985cb60aca6be2175bc512c9243626L76-R78): Added `select-none` and `draggable={false}` attributes to various images in the `Navbar` component. [[1]](diffhunk://#diff-697415f51799297bd64388e550f60789de985cb60aca6be2175bc512c9243626L76-R78) [[2]](diffhunk://#diff-697415f51799297bd64388e550f60789de985cb60aca6be2175bc512c9243626L91-R94) [[3]](diffhunk://#diff-697415f51799297bd64388e550f60789de985cb60aca6be2175bc512c9243626L107-R111) [[4]](diffhunk://#diff-697415f51799297bd64388e550f60789de985cb60aca6be2175bc512c9243626R130-R131) [[5]](diffhunk://#diff-697415f51799297bd64388e550f60789de985cb60aca6be2175bc512c9243626L180-R187) [[6]](diffhunk://#diff-697415f51799297bd64388e550f60789de985cb60aca6be2175bc512c9243626L196-R204) [[7]](diffhunk://#diff-697415f51799297bd64388e550f60789de985cb60aca6be2175bc512c9243626L212-R221)

### Decorative Image Additions:
* [`app/(main)/_components/tracks-section.tsx`](diffhunk://#diff-7ed78e2061636bac27afdd60e336fffd4f689df1be220e64a8ccd10e016e5319R1-R46): Added decorative orange branch images to the `TracksSection` component.

### Text Styling Improvements:
* [`app/(main)/_components/heading.tsx`](diffhunk://#diff-38bcb41921a40de8ae1ed55132363f783274dc56e1bb2df9643e1a6d5c4128b4L53-R53): Added a drop shadow to the main heading text.
* [`app/(main)/_components/mobileheading.tsx`](diffhunk://#diff-3b67c8ea597c94e6578d47fda1996386cbe67124784f36c91b6def5ab92b4523L14-R14): Added a drop shadow to the mobile heading text.
* [`app/(main)/_components/faqs-section.tsx`](diffhunk://#diff-da0ab51f7e03a171ce8c99ceaea66adccac0b1f0b91b460b7f3ab0cf75d37f54L7-R8): Adjusted padding and width of the FAQ section for better layout.

These changes collectively enhance the visual appeal and usability of the website, providing a more polished and engaging user experience.

![image](https://github.com/user-attachments/assets/0aa79b24-3b19-4ae7-bd14-1b7365069171)
![image](https://github.com/user-attachments/assets/ab3a3819-cda1-4d8c-ae05-1448e35d71c1)
![image](https://github.com/user-attachments/assets/0608ee98-f7a0-4ba4-af6c-d936cf5c5725)
![image](https://github.com/user-attachments/assets/379315d1-f0b6-4e5b-95bb-ce452bedeadb)

dtsivkovski commented on app/(main)/_components/mobileheading.tsx:
```diff
@@ -11,7 +11,7 @@ const MobileHeading = () => {
+        <h1 className="flex flex-wrap font-TangoSansBold pantherhacks-title text-5xl sm:text-6xl text-[#befcfd] w-full px-4 sm:px-16 drop-shadow-lg">
           PANTHERHACKS {hackathonDateInfo.year}
         </h1>
         <Image
```
```suggestion
        <Image
          draggable={false}
```

This looks really good, Dylan! Only thing I noticed is I feel like we should make the mobile heading image to not be draggable as well, while we're at it!

dtsivkovski reviewed (approved):
Overall, though, looks good! Feel free to implement that change if you'd like.

DylanDevelops commented:
> Overall, though, looks good! Feel free to implement that change if you'd like.

I implemented this and fixed a few other things since you last checked. Can you rereview?

dtsivkovski commented on app/(main)/_components/track.tsx:
```diff
@@ -4,13 +4,27 @@ import "./track.css";
-const Track: React.FC<ITrackProps> = ({ name, imageSrc }) => {
+const Track: React.FC<ITrackProps> = ({ name }) => {
   return (
     <div className="flex flex-col items-center space-y-4 md:hover:scale-105 transition-transform duration-300 md:cursor-pointer">
```
```suggestion
    <div className="flex flex-col items-center space-y-4 md:hover:scale-105 transition-transform duration-300">
```

Looks good! This is the only other change I'd suggest implementing. Even if it has a hover effect, if something doesn't have an action or link then it shouldn't have a mouse pointer to avoid confusion for the user.

dtsivkovski reviewed (changes requested):
Once this change is implemented we should be good to go!

dtsivkovski reviewed (approved):
Good work! Looks good to merge!
